apps/terminal/models/component/endpoint.py

```diff
@@ -37,17 +38,10 @@ def __str__(self):
         return self.name
 
     def get_port(self, target_instance, protocol):
-        from terminal.utils import db_port_manager
-        from assets.const import DatabaseTypes, Protocol
-
-        if isinstance(target_instance, Asset) and \
-                target_instance.is_type(DatabaseTypes.ORACLE) and \
-                protocol == Protocol.oracle:
-            port = db_port_manager.get_port_by_db(target_instance)
-        else:
-            if protocol in [Protocol.sftp, Protocol.telnet]:
-                protocol = Protocol.ssh
-            port = getattr(self, f'{protocol}_port', 0)
+        from assets.const import Protocol
+        if protocol in [Protocol.sftp, Protocol.telnet]:
+            protocol = Protocol.ssh
+        port = getattr(self, f'{protocol}_port', 0)
         return port
 
     def is_default(self):
```

Comment:
Based on the given code snippet, I found no obvious irregularities or issues to mention. However, you should perform some type of security test if these ports (e.g., PostgreSQL, Redis...) are exposed publicly and ensure proper access controls based on your specific requirements.

For optimization:
Consider using string formatting where possible rather than concatenations for consistency and readability.

Note: This assessment is based on the knowledge cutoff provided and could have limited applicability due to changes that may occur after 2021-09-01 depending on how open-source libraries and frameworks were updated during this period. Please refer to documentation updates released later in case there are significant changes between now and then.